infinispan-spring-boot-samples/remote/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.infinispan</groupId>
         <artifactId>infinispan-spring-boot-starter-parent</artifactId>
-        <version>2.1.8-SNAPSHOT</version>
+        <version>2.2.0-SNAPSHOT</version>
```

Comment:
The master branch in `infinispan-spring-boot-starter-parent' is still pointing to `2.1.8`, right? 
https://github.com/infinispan/infinispan-spring-boot/blob/115d6a6b7dfab0b523610d107985feaf8f69a0e9/pom.xml#L13

Reply:
master yes. this is for the new release 2.2.x